### PR TITLE
(PUP-7914) Document configuring Windows event log

### PR DESCRIPTION
**Pull request by Iristyle**

### CODE REVIEW

File: source/puppet/5.0/services_agent_windows.markdown [hunk 1]
```diff
@@ -143,6 +143,16 @@ Configure Puppet agent with [puppet.conf][], using the `[agent]` and/or `[main]`
 
 When running as a service, Puppet agent logs messages to the Windows Event Log. You can view its logs by browsing the Event Viewer. (Control Panel → System and Security → Administrative Tools → Event Viewer)
 
+By default, Puppet logs to the `Application` event log, but optionally, Puppet may be configured to log to its own `Puppet` log instead. To enable the `Puppet` log requires creating a specific registry key to configure the new event log destination. This can be done using the `reg.exe` tool:
```
Comment (glennsarti) [inline source/puppet/5.0/services_agent_windows.markdown]: `To enable the `Puppet` log requires creating` doesn't read/flow well  Perhaps:

"Use the `reg.exe` tool, shown below, to enable the `Puppet` log"

File: source/puppet/5.0/services_agent_windows.markdown [hunk 2]
```diff
@@ -143,6 +143,16 @@ Configure Puppet agent with [puppet.conf][], using the `[agent]` and/or `[main]`
...
+
+    C:\> reg add HKLM\System\CurrentControlSet\Services\EventLog\Puppet\Puppet /v EventMessageFile /t REG_EXPAND_SZ /d "C:\Program Files\Puppet Labs\Puppet\bin\puppetres.dll"
+
+Alternatively, there is a PowerShell cmdlet `New-EventLog` that may be used to do the same thing:
```
Comment (glennsarti) [inline source/puppet/5.0/services_agent_windows.markdown]: reword to: Alternatively, use the `New-EventLog`  PowerShell cmdlet as below:

File: source/puppet/5.0/services_agent_windows.markdown (hunk 1 above)
Comment (Iristyle) [reply source/puppet/5.0/services_agent_windows.markdown]: Will tweak... though I think the "shown below" is redundant.

**Comment by clairecadman** [comment]
@Iristyle Just noticed the do not merge, do you want me to revert this?

**Comment by gguillotte** [comment]
@clairecadman It's OK, I've got nothing to add here. I just need to also apply these changes to the other relevant versions of Puppet in the docs.

**Pull request merged**